README.md

```diff
@@ -241,92 +241,22 @@ If you use `SWISNAP_ENABLE_<plugin_name>` set to `true`, then keep in mind that
 
 ## Integrating Kubernetes Cluster Events Collection With Loggly
 
-Version 22 of Kubernetes collector allows you to collect cluster events and push them to Loggly using logs collector under the hood. To enable event collection in your deployment, follow below steps:
-* Create `kubernetes.yaml` file that will configure kubernetes collector. This config should contain `collector.kubernetes.all.events` field with specified filter. Following example config will watch for `normal` events in `default` namespace:
-  ```yaml
-  collector:
-    kubernetes:
-      all:
-        incluster: true
-        kubeconfigpath: ""
-        interval: "60s"
-
-        events: |
-          # Embedded YAML (as a multiline string literal)
-          filters:
-          - namespace: default
-            type: normal
-
-        grpc_timeout: 30
-
-  load:
-    plugin: snap-plugin-collector-aokubernetes
-    task: task-aokubernetes.yaml
-  ```
-* If you want to monitor events count in AppOptics, then edit your current `task-aokubernetes.yaml` task manifest so it contains `/kubernetes/events/count` metric in `workflow.collect.metrics` list, and copy it to working directory:
-  ```yaml
-  ---
-  version: 1
-
-  schedule:
-    type: streaming
-
-  deadline: "55s"
-
-  workflow:
-    collect:
-
-      config:
-        /kubernetes:
-          MaxCollectDuration: "2s"
-          MaxMetricsBuffer: 250
-
-      metrics:
-        /kubernetes/events/count: {}
-        /kubernetes/pod/*/*/*/status/phase/Running: {}
-      publish:
-      - plugin_name: publisher-appoptics
-        config:
-          period: 60
-          floor_seconds: 60
-  ```
-* Create `logs.yaml` file configuring the logs collector. Make sure that logs collector looks for `/var/log/SolarWinds/Snap/events.log` file:
-  ```yaml
-  collector:
-    logs:
-      all:
-        loggly_token: <your loggly token>
-        api_host: "logs-01.loggly.com"
-
-        api_port: 514
-        api_protocol: "tcp"
-
-        connect_timeout: "30s"
-
-        write_timeout: "30s"
-
-        files: |
-          /var/log/SolarWinds/Snap/events.log
-
-        exclude_patterns: |
-          .*self-skip-logs-collector.*
+Version 22 of Kubernetes collector allows you to collect cluster events and push them to Loggly using logs collector under the hood. To utilise this functionality there is a need to create corresponding configmaps in your cluster, with proper plugins configration. Those files can be found in  [Event collector configs](examples/event-collector-configs). To enable event collection in your deployment, follow below steps:
```

Comment:
please add a link to documentation page: https://documentation.solarwinds.com/en/Success_Center/appoptics/Content/kb/host_infrastructure/host_agent/kubernetes_ha.htm#integrating-kubernetes-cluster-events-collection-with-loggly

Reply:
Link added